### ISSUE TITLE
Freecam keybind registers when using the gamemode switcher

### ISSUE THREAD

**Issue by JelCraft** [MinecraftFreecam/Freecam#249]
### Description of the issue

When pressing the keybind combination to toggle the gamemode switcher (F3+F4), this also toggles freecam if the default Freecam keybind (F4) is enabled in the config. This only happens when the player has permission to toggle gamemodes.

### Expected behavior

When pressing F3+F4 Freecam should not activate. Even if the default Freecam keybind is F4.

### Steps to reproduce

1. Open a singleplayer world or join a server
2. Press F3+F4 (with /gamemode permissions)
3. The gamemode switcher pops up as expected, but Freecam also activates

### How prevalent is the issue?

Every time the gamemode switcher is used

### Freecam version

1.3.0

### Are you using "Modrinth Edition"?

Yes

### Minecraft version

1.21.0

### Modding platform

- [X] Fabric
- [ ] Neoforge
- [ ] Forge

**Comment by MattSturgeon** [comment]
Hm, it's been a while since I looked at this but I remember specifically having this case in mind when I last worked on the keybind implementation.

Thanks for the report. I'll try and find time to investigate if nobody beats me to it.

**Comment by MattSturgeon** [comment]
Ah, I remember now.

This is a duplicate of #175.

TL;DR:
- If you release `F4` _after_ releasing `F3`, then freecam's keybind is triggered.
- If you release `F4` while _still holding_ `F3`, you should get the intended behaviour.
